src/WebJobs.Script/ScriptConstants.cs

```diff
@@ -0,0 +1,16 @@
+﻿// Copyright (c) .NET Foundation. All rights reserved.
+// Licensed under the MIT License. See License.txt in the project root for license information.
+
+namespace Microsoft.Azure.WebJobs.Script
+{
+    public static class ScriptConstants
+    {
+        public const string AzureFunctionsWebHookContextKey = "MS_AzureFunctionsWebHookContext";
+
+        public const string AzureFunctionsHttpResponseKey = "MS_AzureFunctionsHttpResponse";
+
+        public const string DefaultSystemTriggerParameterName = "<AF>systemTrigger";
```

Comment:
I'm using <> here to guarantee that we won't have collisions with any user defined parameters. Since users won't be able to compile a function with <> in the argument name, we don't have to worry and check if the name collide with anything. I can still rename it to "<AF>_functionTriggerValue" if desired, but it would probably be safer to keep the <AF> prefix. Thoughts?